### PR TITLE
[KHRGA_30] Updated command accessor

### PR DESCRIPTION
**Pull request by imvucic**
Issue: KHRGA_30

### CODE REVIEW

File: source/spelling_wordlist.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ accessor
 accessors
 backend
 backends
+const
```
Comment (PetroBondar) [inline source/spelling_wordlist.txt]: Khronos Guys asked us to keep const as in code formatting.

File: source/iface/command-accessor.rst [hunk 2]
```diff
@@ -6,281 +6,274 @@
...
+    - Access mode
+  * - not const-qualified
+    - ``access_mode::read``
+  * - const-qualified
+    - ``access_mode::read``
 
-  *Available only when:
-   accessTarget == access::target::global_buffer*
+The specializations of ``accessor`` with
+``target::device or target::host_task`` that are read-only accessors.
```
Comment (PetroBondar) [inline source/iface/command-accessor.rst]: "or" in code

File: source/iface/command-accessor.rst [hunk 3]
```diff
@@ -6,281 +6,274 @@
...
+    - Access mode
+  * - not const-qualified
+    - ``access_mode::read_write``
 
-Returns pointer to memory in a host buffer.
+The table present an implicit conversion from
+the read-write specialization.
 
+========================
+Common types and members
```
Comment (PetroBondar) [inline source/iface/command-accessor.rst]: Still believe that we should add common types here. Let's wait for Khronos for this decision

File: source/iface/command-accessor.rst [hunk 4]
```diff
@@ -6,281 +6,274 @@
...
 
-::
+  template <typename AllocatorT, typename TagT>
+  accessor(buffer<DataT, Dimensions, AllocatorT>& bufferRef,
+           handler& commandGroupHandlerRef, range<Dimensions> accessRange,
+           id<Dimensions> accessOffset, TagT tag,
+           const property_list& propList = {})
 
-  constexpr bool is_placeholder() const;
+Constructors of the accessor class.
```
Comment (PetroBondar) [inline source/iface/command-accessor.rst:117]: Maybe it's nice to have a detailed description of constructors in the reference guide. You can take structure from buffer constructors section

File: source/iface/command-accessor.rst [hunk 5]
```diff
@@ -6,281 +6,274 @@
 
 .. _buffer-accessor:
 
-******************
+==================
 ``sycl::accessor``
```
Comment (PetroBondar) [inline source/iface/command-accessor.rst:10]: I need a table with targets from this part and a reference like ``_access-target`` to this table for the buffers chapter

File: source/spelling_wordlist.txt (hunk 1 above)
Comment (imvucic) [reply source/spelling_wordlist.txt]: Removed with https://github.com/imvucic/SYCL_Reference/pull/20/commits/7b6e03dc9cca91697627588f7f7cab406545983a

File: source/iface/command-accessor.rst (hunk 2 above)
Comment (imvucic) [reply source/iface/command-accessor.rst]: fixed https://github.com/imvucic/SYCL_Reference/pull/20/commits/7b6e03dc9cca91697627588f7f7cab406545983a

File: source/iface/command-accessor.rst (hunk 3 above)
Comment (imvucic) [reply source/iface/command-accessor.rst]: Let's wait response from Khronos

File: source/iface/command-accessor.rst (hunk 5 above)
Comment (imvucic) [reply source/iface/command-accessor.rst:10]: Added with https://github.com/imvucic/SYCL_Reference/pull/20/commits/7b6e03dc9cca91697627588f7f7cab406545983a

**Pull request closed without merging**